ansible/kubernetes-spec/jupyterhub/deployment.yml

```diff
@@ -58,39 +58,35 @@ spec:
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: jupyterhub-deployment
+  name: jupyterhub
 spec:
   replicas: 1
   template:
     metadata:
       labels:
         name: jupyterhub-pod
-        confighash: "{{ kubernetes_jupyterhub_config_hash }}"
     spec:
       serviceAccount: jupyterhub-spawner
       containers:
       - name: jupyterhub-container
-        image: manics/jupyterhub-githubauth:merge
+        image: imagedata/jupyterhub-githubauth:latest
```

Comment:
Needs a tag?

Reply:
Looks like it was already tagged, updated.